src/models/worldStateModel.ts

```
@@ -0,0 +1,613 @@
+    LibraryInfo: LibraryInfoSchema,
+    PVPChallengeInstances: [PVPChallengeInstanceSchema],
+    ProjectPct: [Number],
+    EndlessXpChoices: [EndlessXpChoiceSchema],
+    FeaturedGuilds: [FeaturedGuildSchema],
+    SeasonInfo: SeasonInfoSchema,
+    Tmp: String
+});
+
+WorldStateSchema.set("toJSON", {
+    transform(_document, returnedObject) {
+        delete returnedObject.__v;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)